code/modules/power/singularity/emitter.dm

```diff
@@ -256,90 +273,92 @@
 
 	return ..()
 
-/obj/machinery/power/emitter/wrench_act(mob/living/user, obj/item/I)
-	..()
-	default_unfasten_wrench(user, I)
+/obj/machinery/power/emitter/wrench_act(mob/living/user, obj/item/item)
+	. = ..()
```

Comment:
Same comment about parent calls and . here

Reply:
same, just consistency